### PR TITLE
[TOD][Datasets][Easy] Taskmaster2 to TOD Conversations format

### PR DESCRIPTION
**Pull request by moyapchen**
Title. I only include System + UserSimulator Teachers here since that's all we need right now from dataset.

There's a legacy implementation of Taskmaster2 here, but doesn't seem to be anyone using, so clobbering.

---------------------------------

Datasets added in this substack:
* Google SGD
   * Google SGD Simulation Splits (In-domain, Out-domain)
* MetalWoz
* MSR_E2E
* Multidogo
* MultiWoz V2.2
* Taskmaster
* **Taskmaster2**
* Taskmaster3 (TicketTalk)

Test plan:
Regression test, `parlai dd` of dataset

### CODE REVIEW

File: parlai/tasks/taskmaster2/README.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # Taskmaster 2
 
 Originally from the
-[Google Research Datasets](https://github.com/google-research-datasets/Taskmaster/blob/main/TM-2-2020/README.md).
+[Google Research Datasets](https://github.com/google-research-datasets/Taskmaster/blob/master/TM-2-2020/README.md).
```
Comment (moyapchen) [inline parlai/tasks/taskmaster2/README.md:4]: This is indeed the correct path, interestingly enough. Not sure exactly where this changed though hmm...

**Comment by moyapchen** [comment]
NOTE: Need to add this to `task_list.py` in order to fix the broken test - going to do this tomorrow, though this should still be fine to review even without

(Same with lint)

File: parlai/tasks/taskmaster2/README.md (hunk 1 above)
Comment (stephenroller) [reply parlai/tasks/taskmaster2/README.md:4]: Wild to see Google make the opposite push as us.

**Review by stephenroller — APPROVED** [review]
Stamp

File: parlai/tasks/taskmaster2/README.md (hunk 1 above)
Comment (moyapchen) [reply parlai/tasks/taskmaster2/README.md:4]: Looking at the history of that file, I think the `master->main` change came in from a regex from when we did that. (No intentional moving from `main->master` on their part.

**Pull request merged**